cloudbrain/rt_server/rt_server.py

```diff
@@ -16,7 +17,19 @@
 SERVER_PORT = 31415
 logging.getLogger().setLevel(logging.INFO)
 
+def get_args_parser():
+    parser = argparse.ArgumentParser()
 
+    parser.add_argument('-c', '--cloudbrain', default=RABBITMQ_ADDRESS,
+                        help="The address of the RabbitMQ instance you are sending data to.\n"
```

Comment:
A note for the future: in our codebase, we try to limit the number of characters per line to 80 characters per line. The reason for that is that we follow the PEP-8 guidelines. See https://www.python.org/dev/peps/pep-0008/#maximum-line-length

Obviously, you are not going to count the number of chats in each line manually every time :smile: So the best is to setup you editor to be PEP-8 compliant (here's a post about pycharm http://blog.jetbrains.com/pycharm/2013/02/long-awaited-pep-8-checks-on-the-fly-improved-doctest-support-and-more-in-pycharm-2-7/). Lookup what your preferred code editor has in term of linting plugins.